### ISSUE TITLE
Help needed?

### ISSUE THREAD

**Issue by imjasonmiller** [stormalinblue/gulp-mathjaxify#1]
Hi!

This package looks great. Is there anything that is preventing this package from being published that I could perhaps help with?

```sh
npm i https://github.com/stormalinblue/gulp-mathjaxify.git
```
Installing the repository in the above manner doesn't expose the [`lib`](https://github.com/stormalinblue/gulp-mathjaxify/blob/2ad67a16ba28465fe3a3baf6534095ce5f7bd66c/.npmignore#L82) folder, making it, as far as I know, impractical to install locally and on other systems?

**Comment by stormalinblue** [comment]
The `lib` folder only contains TypeScript files, which cannot be directly run without compilation. I have set up Travis to deploy the compiled files to npm, where it can be downloaded from and used.

This project was delayed because I found that `gulp-mathjax-page` served my needs temporarily, but uses an older version of `gulp-mathjax-node` than is currently available (see my issue [here](https://github.com/roman-spiridonov/gulp-mathjax-page/issues/1#issue-451642407)).
